frontend/amundsen_application/static/js/pages/TableDetailPage/LineageList/index.tsx

```diff
@@ -55,8 +69,19 @@ export const LineageList: React.FC<LineageListProps> = ({
     );
   }
 
-  return (
+  const message = AppConfig.tableLineage.inAppListMessageGenerator &&
+    tableDetails &&
+      AppConfig.tableLineage.inAppListMessageGenerator(
```

Comment:
I think you can make `AppConfig.tableLineage.inAppListMessageGenerator?.(...` instead